docs/brisket.createserver.md

```diff
@@ -83,6 +83,24 @@ var brisketServer = Brisket.createServer({
 });
 ```
 
+#### environmentConfig.clientAppBootstrapFunctionName
+`environmentConfig.clientAppBootstrapFunctionName` is an optional string for defining a function to bootstrap ClientApp. By default, Brisket will bootstrap ClientApp synchronously at the bottom of the body. By defining a `clientAppBootstrapFunctionName`, Brisket will create a function at `window[clientAppBootstrapFunctionName]` so that the application can decide when the bootstrapping starts.
+
+```js
+var brisketServer = Brisket.createServer({
+    apiHost: 'http://localhost:4000',
+    clientAppRequirePath: 'app/ClientApp',
+
+    ServerApp: ServerApp,
+    environmentConfig: { appRoot: '/path/to/app/', clientAppBootstrapFunctionName: 'startBrisket' }
+});
+
+// In the browser
```

Comment:
Something I didn't consider when writing this documentation is that my above sample is disingenuous. Since Brisket appends this script to the end of the body, synchronously calling `window.startBriske()t` would always fail since the actual setup of this function would always be after it. 

Any suggestions for how I can communicate this?